### PR TITLE
Feature: Expose Account Timing in GraphQL

### PR DESCRIPTION
**Pull request by MartinMinkov**
This PR exposes the `Account_timing` information on an account in our GraphQL. 

This was tested by using a ledger that had time-locked accounts enabled as well as normal accounts. Manual queries were done to ensure correctness. 

**Timed Account**
image

**Normal Account**
image

### CODE REVIEW

File: src/lib/mina_graphql/mina_graphql.ml [hunk 1]
```diff
@@ -346,6 +346,57 @@ module Types = struct
             ~resolve:(fun _ {Fee_transfer.fee; _} -> Currency.Fee.to_uint64 fee)
         ] )
 
+  let account_timing : (Mina_lib.t, Account_timing.t option) typ =
+    obj "AccountTiming" ~fields:(fun _ ->
+        [ field "timed_minimum_balance" ~typ:uint64
```
Comment (psteckler) [inline src/lib/mina_graphql/mina_graphql.ml:351]: actually, I think this can be `initial_minimum_balance`

File: graphql_schema.json [hunk 2]
```diff
@@ -6582,6 +6582,87 @@
...
+          "possibleTypes": null
+        },
+        {
+          "kind": "OBJECT",
+          "name": "AccountTiming",
+          "description": null,
+          "fields": [
+            {
+              "name": "timed_minimum_balance",
+              "description": "The timed minimum balance for an account",
```
Comment (psteckler) [inline graphql_schema.json:6602]: could be `initial_mininum_balance`

**Review by psteckler — APPROVED** [review]

**Comment by psteckler** [comment]
Should mention in PR how this was tested.

**Review by psteckler — APPROVED** [review]

File: src/lib/mina_graphql/mina_graphql.ml (hunk 1 above)
Comment (MartinMinkov) [reply src/lib/mina_graphql/mina_graphql.ml:351]: sounds good!

**Comment by MartinMinkov** [comment]
This PR was moved too https://github.com/MinaProtocol/mina/pull/7746

**Pull request closed without merging**